docker/compose.yml

```
@@ -0,0 +1,21 @@
+services:
+  tuono:
+    container_name: tuono-source-container
+    hostname: tuono-source-container
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile
+      args:
+        RUST_VERSION: 1.83
+        BASE_OS: bookworm
+        NODE_VERSION: 22.11.0
+    image: tuono-source-image
+    ports:
+      - "3000:3000" # for apps/documentation frontend app
```

Actually this is incorrect:
someone could one of the site inside `examples` folder

```suggestion
      - "3000:3000" # expose default port used by tuono server
```

---

yes indeed !